src/test/java/school/hei/haapi/integration/PaginationIT.java

```diff
@@ -67,21 +64,22 @@ private void createNewStudents(int nbOfNewStudents) throws ApiException {
 
   @Test
   void student_pages_are_ordered_by_reference() throws ApiException {
-    createNewStudents(100);
+    createNewStudents(10);
+    int pageSize = 4;
     ApiClient teacher1Client = anApiClient(TEACHER1_TOKEN);
-
     UsersApi api = new UsersApi(teacher1Client);
-    List<Student> page5 = api.getStudents(5, 5);
-    List<Student> page10 = api.getStudents(10, 5);
-    List<Student> page100 = api.getStudents(100, 5);
 
-    assertEquals(5, page5.size());
-    assertEquals(5, page10.size());
+    List<Student> page1 = api.getStudents(1, pageSize);
+    List<Student> page2 = api.getStudents(2, pageSize);
+    List<Student> page100 = api.getStudents(100, pageSize);
+
+    assertEquals(pageSize, page1.size());
+    assertEquals(pageSize, page2.size());
```

Comment:
Also get page3 and tests that its size equals 2

Reply:
Done

Reply:
Actually, page3 size equals 4 not 2 ...

Reply:
It was because we have already created 2 students in the database